Contribute/dotnet/dotnet-contribute.md

```diff
@@ -149,13 +152,12 @@ Samples are full programs and libraries meant for download. They may be small in
 3. Your sample should include **appropriate exception handling**. It should handle all exceptions that are likely to be thrown in the context of the sample. For example, a sample that calls the [Console.ReadLine](https://docs.microsoft.com/dotnet/api/system.console.readline) method to retrieve user input should use appropriate exception handling when the input string is passed as an argument to a method. Similarly, if your sample expects a method call to fail, the resulting exception must be handled. Always handle the specific exceptions thrown by the method, rather than base class exceptions such as [Exception](https://docs.microsoft.com/dotnet/api/system.exception) or [SystemException](https://docs.microsoft.com/dotnet/api/system.systemexception).
 
 4. If your sample builds a standalone package, you must include the runtimes used by our CI build system, in addition to any runtimes used by your sample:
+
     - `win7-x64`
     - `win8-x64`
     - `win81-x64`
     - `ubuntu.16.04-x64`
 
-We will have a CI system in place to build these projects shortly.
```

Comment:
@adegeo, I deleted this sentence as I think it's outdated now. Can you confirm if you want to delete it?

Reply:
I think the runtime list above also needs to change to win10-x64 only as our CI system is currently only that.

Reply:
@adegeo I find the whole sentence isn't clear to me a bit.

> 4. If your sample builds a standalone package, you must include the runtimes used by our CI build system, in addition to any runtimes used by your sample:
>
>    - `win7-x64`
>    - `win8-x64`
>    - `win81-x64`
>    - `ubuntu.16.04-x64`

If the list needs to be changed, I think it's worth to make the sentence more clear. Do you have any suggestions?

Reply:
> it's worth to make the sentence more clear. 

I agree

Reply:
I'm not exactly sure what t is trying to communicate. We should find the original author

Reply:
@BillWagner can you help clarify?

Reply:
To match what we do now, we should remove the entire list item (4 in the article). We don't build on multiple platforms, so we don't need to specify which platforms to build packages for.